src/main/java/org/sonarsource/sonarlint/ls/SonarLintExtendedLanguageServer.java

```diff
@@ -217,11 +217,9 @@ public void setConnectionId(String connectionId) {
 
   class GetServerPathForTokenGenerationResponse {
     private String serverUrl;
-    private  String errorMessage;
 
-    public GetServerPathForTokenGenerationResponse(String serverUrl, String errorMessage) {
+    public GetServerPathForTokenGenerationResponse(String serverUrl) {
```

Comment:
Why was errorMessage removed? Will exceptions be just thrown if something goes wrong?
Client will also need to be updated so that it does not expect `errorMessage` parameter anymore 👍

Reply:
Error message removed because now we pass Future directly and there is no need in it. Future will be either resolved or failed. Good point about client.